### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

File: doc/firehol/firehol-mac.5.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The `mac` helper command DROPs traffic from the *IP* address that was not
 sent using the *macaddr* specified.
 
 When packets are dropped, a log is produced with the label "MAC
-MISSMATCH" (sic.). `mac` obeys the default log limits (see
+MISMATCH" (sic.). `mac` obeys the default log limits (see
```
Comment (philwhineray) [inline doc/firehol/firehol-mac.5.md:33]: Unfortunately I don't think we should take this particular fix, as if someone is matching this values in their logs we will break their setup.

The (sic.) indicates we know this is a mistake, so if we were to fix it, we'd want to remove that at the same time, but I don't think we can.

File: sbin/firehol [hunk 2]
```diff
@@ -4358,10 +4358,10 @@ mac() {
 	if running_ipv6; then
 		if [ ${wrongmac6_chain} -eq 0 ]
 		then
-			set_work_function "Creating the MAC-MISSMATCH chain (only once)"
+			set_work_function "Creating the MAC-MISMATCH chain (only once)"
 
 			ip6tables -t filter -N WRONGMAC
-			rule table filter chain WRONGMAC loglimit "BLOCKED MAC MISSMATCH" action DROP || return 1
+			rule table filter chain WRONGMAC loglimit "BLOCKED MAC MISMATCH" action DROP || return 1
```
Comment (philwhineray) [inline sbin/firehol:4364]: We can't really change this in case someone is depending on the current behaviour.

**Review by philwhineray — CHANGES_REQUESTED** [review]
Thanks for taking the time to improve firehol!

I don't think we can take the MISSMATCH fixes, as they will change the interoperability with log filescanners and the like.

Other than that, this looks great.

File: doc/firehol/firehol-mac.5.md (hunk 1 above)
Comment (rex4539) [reply doc/firehol/firehol-mac.5.md]: Reverted.

File: sbin/firehol (hunk 2 above)
Comment (rex4539) [reply sbin/firehol]: Reverted.

**Review by philwhineray — APPROVED** [review]

**Comment by philwhineray** [comment]
Great, thanks again!

**Pull request merged**